docs/en/guide/automation/pattern-events.md

```diff
@@ -11,16 +11,15 @@ Check out the [advanced examples](../firmware/stock-firmware#advanced-examples)
 
 ## Doorbell Pattern
 
+### Event (Sensors)
```

Comment:
You renamed the pattern event entites withoud appending `_pattern`

Reply:
Updated